docs/docs/sourcing-from-buttercms.md

```diff
@@ -95,31 +95,25 @@ Thumbnail URL = https://fs.buttercms.com/resize=width:200,height:200/zjypya5tRny
 
 Resizing is just one of the many different transformations you can do to your images. Refer to the [Filestack docs](https://www.filestack.com/docs/) for full details.
 
-## Localization
+### Localization
 
 ButterCMS has full support for localization of your content. Locale names and keys are completely customizable and there's no limit to the number of locales you can have. View their [API Reference](https://buttercms.com/docs/api/) to learn how to query by locale.
 
 ![locales](https://buttercms.com/static/images/docs/guides/Localization.png)
 
-# Creating pages
+## Creating pages
 
-### Introduction
-
-Quickly launch a new marketing site or add CMS-powered pages to your existing site using our Pages.
+### Creating a single page(home page)
 
-## Creating a single page(home page)
-
-### Introduction
+#### Introduction
 
 Quickly launch a new marketing site or add [CMS-powered pages](https://buttercms.com/gatsbyjs-cms/) to your existing site using our Pages.
 
-### Create a single Page
-
 Adding a CMS-powered page to your app involves three easy steps:
 
 1. Create the Page structure
-1. Populate the content
-1. Integrate into your application
+2. Populate the content
```

Comment:
Markdown increments list items automatically, so these shouldn't need to change.